spec/elastic_apm/normalizers/active_record_spec.rb

```diff
@@ -90,6 +90,58 @@ def normalize_payload(payload)
             )
             expect(subtype).to eq 'postgres'
           end
+          
+          it 'resolves the connection_id object id to a connection if the full connection is missing' do
+            sql = 'SELECT  "burgers".* FROM "burgers" ' \
+              'WHERE "burgers"."cheese" = $1 LIMIT 1'
+
+            _name, _type, subtype, = normalize_payload(
+              sql: sql,
+              connection_id: double(adapter_name: 'MySQL').object_id
+            )
+            expect(subtype).to eq 'mysql'
+
+            _name, _type, subtype, = normalize_payload(
+              sql: sql,
+              connection_id: double(adapter_name: 'Postgres').object_id
+            )
+            expect(subtype).to eq 'postgres'
```

Comment:
What is the reason for testing this twice with a different `adapter_name`?

Reply:
Pretty much only because the comparable pre-existing test for using the `:connection` value checked both. The only assumption I might be able to make about the original tester's intent was to check that the adapter name gets downcased properly?

See this existing test on `master`: https://github.com/elastic/apm-agent-ruby/blob/master/spec/elastic_apm/normalizers/active_record_spec.rb#L77-L92

Reply:
Thanks for the explanation. I don't know why exactly we have more than one check for downcasing the adapter name. I guess we can just skip doing a second check in your tests.